common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -189,6 +191,7 @@ public static void submitSilent(@SuppressWarnings(WarningType.rawtype_warning) @
     public static FinalizableResultFuture<CommandResult> submitSilentReturningFuture(@SuppressWarnings(WarningType.rawtype_warning)
                                                                                          @NonNull final BaseCommand command) {
 
+        CodeMarkerManager.getInstance().markCode(PerfConstants.CodeMarkerConstants.ACQUIRE_TOKEN_SILENT_START);
```

Comment:
Rather than calling `getInstance()` at different spots inside the code, would it make sense for this class to have a single reference to it? Or, at least for a method to declare it / access it once?

I would say that if we're in a lexical that allows us to declare it once, that's preferable that calling the getter multiple times